src/qtiItem/core/Element.js

```diff
@@ -187,11 +187,11 @@ var Element = Class.extend({
         if (typeof this.initObject === 'function') {
             append(this.getObject());
         }
-        _.each(this.metaData, function(v) {
+        _.each(this.metaData, function (v) {
             if (Element.isA(v, '_container')) {
                 append(v);
             } else if (_.isArray(v)) {
-                _.each(v, function(v) {
+                _.each(v, function (v) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
                _.forEach(v, function (v) {
```